rpcs3/Emu/System.cpp

```diff
@@ -1865,6 +1867,121 @@ void Emulator::Stop(bool restart)
 	}
 }
 
+std::string Emulator::FormatTitle(double fps) const
+{
+	// Get version by substringing VersionNumber-buildnumber-commithash to get just the part before the dash
+	std::string version = rpcs3::get_version().to_string();
+
+	version = version.substr(0 , version.find_last_of('-'));
+
+	// Add branch and commit hash to version on frame unless it's master.
+	if ((rpcs3::get_branch().compare("master") != 0) && (rpcs3::get_branch().compare("HEAD") != 0))
+	{
+		version = version + "-" + rpcs3::get_version().to_string().substr((rpcs3::get_version().to_string().find_last_of('-') + 1), 8) + "-" + rpcs3::get_branch();
```

Comment:
```suggestion
		version = version + "-" + version.substr((version.find_last_of('-') + 1), 8) + "-" + rpcs3::get_branch();
```
Seems easier to read this way.

Reply:
I just copied old code, didn't look much in it.